src/main/java/net/tiagofar78/prisonescape/game/PrisonEscapeGame.java

```diff
@@ -77,6 +77,32 @@ public int playerJoin(String playerName) {
 		BukkitTeleporter.teleport(player, _prison.getWaitingLobbyLocation());
 		return 0;
 	}
+
+	/**
+	* @return      0 if success<br> 
+	* 				-1 if already on game<br>
+	* 				-2 if player never on game<br>
+	*/
+	public int playerRejoin(String playerName) {
+		if (isPlayerOnGame(playerName)) {
+			return -1;
+		}
+
```

Comment:
I think it would also be good if a different message is sent to players that try to use rejoin when the game is not started. They will already receive a message saying that they were not in the game, but I think it would be nice if there was a specific message for this scenario so players can be told to use join command instead.